UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/CirculatorySystemBase.cs

```diff
@@ -352,7 +356,7 @@ public void NutrimentCalculation(float HeartEfficiency)
 
 		public void ToxinGeneration(float HeartEfficiency)
 		{
-			float Multiplier = 1;
+			float Multiplier = 1 * HeartEfficiency;
```

Comment:
wouldn't be the same just using ``HeartEfficiency``, since it is being multiplied by 1

Reply:
True